src/sentry/options/defaults.py

```diff
@@ -2718,3 +2718,18 @@
     default=10000,
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "celery_split_queue_legacy_mode",
+    default=["post_process_transactions"],
+    flags=FLAG_AUTOMATOR_MODIFIABLE,
+)
+register(
+    "celery_split_queue_rollout",
+    default={"post_process_transactions": 1.0},
+    flags=FLAG_AUTOMATOR_MODIFIABLE,
+)
+register(
+    "celery_split_queue_task_rollout",
+    default={},
+    flags=FLAG_AUTOMATOR_MODIFIABLE,
+)
```

Comment:
This per queue rollout option allows us to do a backward compatible rollout of a new queue without having to issue and deploy 2 sentry PRs.
We could have relied entirely on the settings to enable the split queue rollout, that would have required this procedure:
1. PR1: (sentry)declare the queues in sentry settings
2. PR2: change the arguments of the workers to listen to the new queues 
3. PR3: (sentry) change sentry settings adding the split config configuration.

This way we can replace PR3 with a simple options autoamtor change, which is much quicker and can be rolled out per region.